### PR TITLE
Bug 1698934 - Add the `uninstall-deletion` ping

### PR DESCRIPTION
**Pull request by Dexterp37**
This will be sent by the offboarding cloud function in order to make sure a data deletion request gets sent.

See [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1698934) for more context.

Checklist for reviewer:

- [ ] The description should reference a bug or github issue, if relevant.
- [ ] There must be a [`CHANGELOG.md`](./CHANGELOG.md) entry for any non-test change.
- [ ] Any change to the NPM commands must be carefully reviewed to make sure it won't break the Add-ons pipeline.
- [ ] Any version increase must follow the [release process](./docs/RELEASE_PROCESS.md).

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -134,6 +135,26 @@ In addition to those built-in metrics, the following metrics are added to the pi
 | rally.id |[uuid](https://mozilla.github.io/glean/book/user/metrics/uuid.html) |The id of the Rally client.  |[mozilla-rally/rally-core-addon#505](https://github.com/mozilla-rally/rally-core-addon/pull/505#issuecomment-815826426)||never | |
 | unenrollment.study_id |[string](https://mozilla.github.io/glean/book/user/metrics/string.html) |The id of the study user has left.  |[Bug 1646151](https://bugzilla.mozilla.org/show_bug.cgi?id=1646151#c32)||never | |
 
+## uninstall-deletion
+
+This ping is sent when user clicks the Join button for a study
+and accepts the study policy.
```
Comment (acmiyaguchi) [inline docs/metrics.md:141]: This doesn't look like the correct conditions for the ping.

File: pings.yaml [hunk 2]
```diff
@@ -61,3 +61,19 @@ demographics:
     - https://github.com/mozilla-rally/rally-core-addon/pull/139#issuecomment-736024232
   notification_emails:
     - than@mozilla.com
+
+uninstall-deletion:
```
Comment (acmiyaguchi) [inline pings.yaml:65]: Will this generate a `metrics.uuid.rally.id` field? The new decoder requires enveloped messages to contain this field (https://github.com/mozilla/gcp-ingestion/blob/caf1fcb5aec06f40ab8ee1c67a33df0abe29247c/ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/rally/DecryptRallyPayloads.java#L208-L213), and I want to make sure it doesn't get thrown into additional_properties.

File: docs/metrics.md (hunk 1 above)
Comment (Dexterp37) [reply docs/metrics.md]: Whoops, forgot to commit the updated docs.

File: pings.yaml (hunk 2 above)
Comment (Dexterp37) [reply pings.yaml:65]: good catch, I forgot to mention this in the `metrics.yaml` file.

**Review by rhelmer — APPROVED** [review]
lgtm, Anthony's review is more valuable here.

**Review by acmiyaguchi — APPROVED** [review]

**Pull request merged**